erpnext_com/www/docs/user/manual/en/introduction/do-i-need-an-erp.md

```diff
@@ -16,7 +16,7 @@ traditional accounting as well as ERP applications.
 
   * $$$ - Saves money.
   * **Easier to configure:** Big ERPs are notoriously hard to setup and will ask you a zillion questions before you can do something meaningful.
-  * **Easier to use:** Modern web like user interface will keep your users happy and in familiar territory.
+  * **Easier to use:** Modern web-like-user interface will keep your users happy and in a familiar territory.
```

Comment:
should be `web-like user interface` and not `web-like-user interface`

Reply:
+1
also remove 'a' here, unnecessary